mono/metadata/w32subset.h

```diff
@@ -210,3 +210,24 @@
 #define HAVE_API_SUPPORT_WIN32_IS_WOW64_PROCESS 0
 #endif
 #endif
+
+#ifndef HAVE_LOADLIBRARY
+#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)
+#define HAVE_LOADLIBRARY 1
+#else
+#define HAVE_LOADLIBRARY 0
```

Comment:
LoadLibrary is coming to UWP so we can fix this.

Reply:
And it is an immediately effective policy, nothing technical, can be fixed right away.